src/wp-includes/post.php

```diff
@@ -417,6 +426,7 @@ function create_initial_post_types() {
 			'description'           => __( 'Template parts to include in your templates.' ),
 			'public'                => false,
 			'_builtin'              => true, /* internal use only. don't use this when registering your own post type. */
+			'_edit_link'            => $template_edit_link, /* internal use only. don't use this when registering your own post type. */
```

Comment:
Could we add and edit link to the custom global styles post as well? 

I don't think we need any URL params aside from `canvas`, e.g.,  `/site-editor.php?canvas=edit`

See: https://github.com/WordPress/wordpress-develop/pull/4326/files#diff-ef9c5475561d459917dac57d21e08a6dfeb36f8b1c44b978d3696aec8b97c405R462